opentracing-api/src/main/java/io/opentracing/tag/Tags.java

```diff
@@ -36,6 +36,16 @@ private Tags() {
     public static final String SPAN_KIND_CLIENT = "client";
 
     /**
+     * A constant for setting the span kind to indicate that it represents a producer span.
```

Comment:
nit: other comments skip the "A constant for setting ..." part; let's model after the others in this file.